### PR TITLE
Add EIP: Retroactively Included EIPs

### PR DESCRIPTION
**Pull request by timbeiko**

### CODE REVIEW

File: EIPS/eip-meta-retro.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-meta-retro.md:1]:
```suggestion
---
eip: 7675
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

File: EIPS/eip-7675.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-7675.md:1]:
```suggestion
---
eip: 7675
```
Looks like you changed the filename but didn't add the assigned EIP number.

Comment (timbeiko) [reply EIPS/eip-7675.md:1]: Thanks @abcoathup !

**Comment by lightclient** [comment]
Will this ever reach final? Or do you plan to continue maintaining this list forever? Lists usually feel like they should live in `ethereum/execution-specs`.

**Comment by timbeiko** [comment]
@lightclient i was planning to make it a Living EIP, given it may eventually have CL changes too.

**Comment by timbeiko** [comment]
@lightclient plz

**Review by lightclient — APPROVED** [review]

**Pull request merged**